UnityProject/Assets/Scripts/Objects/Cargo/ConveyorBelts/ConveyorBelt.cs

```diff
@@ -285,8 +290,29 @@ private void ChangeDirection()
 		#region Multitool Interaction
 
 		public MultitoolConnectionType ConType => MultitoolConnectionType.Conveyor;
-		public bool MultiMaster => true;
-		int IMultitoolMasterable.MaxDistance => int.MaxValue;
+
+		public IMultitoolMasterable Master { get; set; }
```

Comment:
i will eat your kidneys

PUT THE FIELDS AND PROPERTIES AT THE TOP

Reply:
it's grouped with the multitool interaction code

Reply:
put it at the top

Reply:
I would have to get every single one, and I've messed around it way too much time to get syncvars to work

Reply:
I don't know what that has to do with organizing code, but if there is actual glue somewhere in our codebase that breaks these if they're not within an interface's region; then that's the only time I will let something like this slide.

Please keep all variables at the top of the class before any functions, it makes navigation easier and isn't messy.

Reply:
What should we do with code that's just being worked on Being added on?